test/inductor/test_minifier.py

```diff
@@ -25,62 +25,63 @@ def _gen_codegen_fn_patch_code(self, device, bug_type):
 
     # Test that compile and accuracy errors after aot can be repro'd (both CPU and CUDA)
     def _test_after_aot(self, device, bug_type, repro_level):
+        # NB: The program is intentionally quite simple, just enough to
+        # trigger one minification step, no more (dedicated minifier tests
+        # should exercise minifier only)
         run_code = textwrap.dedent(
             f"""\
             @torch.compile()
             def inner(x):
-                for _ in range(3):
-                    x = torch.sin(x)
                 x = torch.relu(x)
-                for _ in range(3):
-                    x = torch.cos(x)
+                x = torch.cos(x)
                 return x
 
             inner(torch.randn(20, 20).to("{device}"))
         """
         )
+        isolate = False
+        if device == "cuda" and bug_type == "runtime_error":
+            # This config triggers a CUDA assert error which will
+            # trash the process
+            isolate = True
         patch_code = self._gen_codegen_fn_patch_code(device, bug_type)
         self.assertIsNotNone(patch_code)
         test_proc, _, repro_proc = self._run_full_test_nocode(
-            run_code, "aot", repro_level, patch_code
+            run_code, "aot", repro_level, patch_code, isolate=isolate
```

Comment:
nit: 1 liner `isolate=((device == "cuda") and (bug_type == "runtime_error"))

Reply:
obsolete with recent changes